docs/oscar_references.bib

```diff
@@ -724,3 +724,24 @@ @Book{Zie95
   doi           = {10.1007/978-1-4613-8431-1},
   url           = {https://doi.org/10.1007/978-1-4613-8431-1}
 }
+
+@book{Koz08,
+    AUTHOR = {Kozlov, Dmitry},
+     TITLE = {Combinatorial algebraic topology},
+    SERIES = {Algorithms and Computation in Mathematics},
+    VOLUME = {21},
+ PUBLISHER = {Springer, Berlin},
+      YEAR = {2008},
+       DOI = {10.1007/978-3-540-71962-5},
+       URL = {https://doi.org/10.1007/978-3-540-71962-5},
+}
+
+@book{BH09,
+    AUTHOR = {Bruns, Winfried and Herzog, Jürgen},
+     TITLE = {Cohen-{M}acaulay rings},
+    SERIES = {Cambridge Studies in Advanced Mathematics},
+    VOLUME = {39},
+ PUBLISHER = {Cambridge University Press, Cambridge},
+      YEAR = {2009},
+      NOTE = {2nd edition},
+}
```

Comment:
You will probably have to run bibtool for incorporating your changes, as in `docs/README.md`:
```
bibtool docs/oscar_references.bib -o docs/oscar_references.bib
```

Reply:
Done.